gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Mar 16 20:09:30 PDT 2016
```

Comment:
This file shouldn't be changed (the gradle distribution is downloaded every ci build should be binary only)